### ISSUE TITLE
rabbitmq user's shell is not /bin/false on CentOS 7

### ISSUE THREAD

**Issue by lukebakken** [rabbitmq/rabbitmq-server-release#46]
This is just after a fresh install of `3.6.11`:

```
[root@localhost ~]# fgrep rabbitmq /etc/passwd
rabbitmq:x:994:991:RabbitMQ messaging server:/var/lib/rabbitmq:/bin/bash
```

**Comment by roeigo-stratoscale** [comment]
Any update on this?

**Comment by lukebakken** [comment]
Commit 17332c84081c2b380ba5eb8f93a929e650fc2b1c fixed this issue and has shipped since version `3.7.0`
